CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## [3.0.19] - 2023.09.07
+### Added
+ - Enable data-api for Horizen EON
```

Comment:
Can you be more specific here? Use more then 1 sentence to describe what is available and how - it will be visible in the release notes etc.

Reply:
Sure, will update it